### PR TITLE
test: remove string + int warning on udp-pummel

### PR DESCRIPTION
**Pull request by juanarbol**

### CODE REVIEW

File: test/benchmark-udp-pummel.c [hunk 1]
```diff
@@ -145,6 +145,14 @@ static void timeout_cb(uv_timer_t* timer) {
 }
 
 
+char* build_expected_buf(int number) {
+  char* str = NULL;
+  strcpy(str, EXPECTED);
+  strcpy(str, (char*)&number);
```
Comment (bnoordhuis) [inline test/benchmark-udp-pummel.c:151]: This isn't equivalent to `EXPECTED + number` - in fact, it's UB.

`&EXPECTED[number]` is probably what you're looking for to silence the compiler warning?

Comment (juanarbol) [reply test/benchmark-udp-pummel.c:151]: That's true, and it makes this PR simpler, thank you!

**Review by bnoordhuis — APPROVED** [review]
Thanks!

**Review by santigimeno — APPROVED** [review]

**Comment by santigimeno** [comment]
Landed in https://github.com/libuv/libuv/commit/73084d5604690e782c8196d91b625bf5693c83ce. Thanks!

**Pull request closed without merging**